deploy/charts/cert-manager/cainjector/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: cainjector
 apiVersion: v1
 # The version and appVersion fields are set automatically by the release tool
-version: v0.1.0
+version: v0.2.0
```

Comment:
Can you revert these changes? We set version numbers as part of our release tool, so this file shouldn't really need to be edited 😄

Reply:
(same goes for all Chart.yaml files in this PR)

Reply:
done